docker/data-science/notebooks/aips.py

```diff
@@ -13,25 +16,31 @@
 AIPS_ZK_PORT= os.getenv('AIPS_ZK_PORT') or '2181'
```

Comment:
I think `AIPS_SOLR_HOST` and AIPS_ZK_PORT` can be moved into solr.py. I don't think anything uses ZK in the codebase except for Solr.